apps/rstudio/values.yaml

```diff
@@ -36,4 +46,12 @@ securityContext:
     drop:
       - all
 
-flavor: {}
+flavor:
+  limits:
+    cpu: 1
+    ephemeral-storage: 800Mi
+    memory: 1Gi
+  requests:
+    cpu: 500m
+    ephemeral-storage: 500Mi
+    memory: 500Mi
```

Comment:
Why doesn't flavors have gpu requests and limits?

Reply:
I think it was because by default we don't want to use this setting (have gpus), and only use it if a flavor has set this in the DB manually by an admin. I think the tests (in actions) might use these setting as well to test the charts work.